src/main/scala/com/microsoft/hyperspace/util/HyperspaceConf.scala

```diff
@@ -25,6 +25,22 @@ import com.microsoft.hyperspace.index.IndexConstants
  * Helper class to extract Hyperspace-related configs from SparkSession.
  */
 object HyperspaceConf {
+
+  /**
+   * Returns the config value whether hyperspace is enabled or not.
+   */
+  def hyperspaceApplyEnabled(spark: SparkSession): Boolean = {
+    spark.conf
+      .get(
+        IndexConstants.HYPERSPACE_APPLY_ENABLED,
+        IndexConstants.HYPERSPACE_APPLY_ENABLED_DEFAULT)
+      .toBoolean
+  }
+
+  def setHyperspaceApplyEnabled(spark: SparkSession, apply: Boolean): Unit = {
```

Comment:
In HyperspaceConf, all functions were getter functions. Is it intended?
Since I needed a getter function somewhere, I made it here.

Reply:
It's usually set by user so there is no setter function here so far.
I'm okay with this location or even not using setter.